openapi/openapi.yaml

```diff
@@ -1052,6 +1052,13 @@ components:
           type: string
         logo:
           type: string
+        contact:
```

Comment:
When we update the `openapi.yaml` file we should also update the version number, which will also release a new version of the API client (used in `explainaboard_client`)